### ISSUE TITLE
Typescript issue - "Argument of type 'unknown' is not assignable ..."

### ISSUE THREAD

**Issue by erikpena** [muxinc/strapi-plugin-mux-video-uploader#18]
# Summary
When installing the plugin via npm or yarn, an error is experienced—

```
> tsc

admin/src/containers/HomePage/index.tsx:56:22 - error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'SetStateAction<string | undefined>'.
  Type 'unknown' is not assignable to type '(prevState: string | undefined) => string | undefined'.

56       setUploadError(error);



Found 1 error.
```

# Repro steps
- Create a new installation of Strapi
- Go through the steps to set Strapi up to a point where you can log into it
- Shutdown Strapi
- Using _either_ `npm` or `yarn`, install the `strapi-plugin-mux-video-uploader`
- Observe the result

# Expected behavior
The plugin should install successfully with the available `.ts` and `.tsx` files in the plugin being transpiled to `.js` files (this is what Strapi needs to identify the plugin).

# Actual behavior
The package manager installation does not complete successfully and throws the error described in the "Summary" section.

<img width="1540" alt="Screenshot 2021-08-31 at 15 27 58" src="https://user-images.githubusercontent.com/2187347/131543701-e90bc1b0-734f-4029-ab22-d38e773e6b6b.png">

**Comment by erikpena** [comment]
In troubleshooting this with @tbobkermux, it looks like Typescript has released a new version (4.4.2) as of 5 days ago and that contains tighter validations around typings for errors from try/catch clauses.

This will be an easy fix, but I think the prudent thing, in this case, is to freeze the version of typescript and update intentionally over time.  So, I'll implement a fix to support 4.4.2, but will also set the version in the `package.json` to statically be 4.4.2.

**Comment by erikpena** [comment]
Fix released in v1.2.1
